src/main/java/com/bgsoftware/superiorskyblock/listener/IslandFlagsListener.java

```diff
@@ -82,6 +90,14 @@ private void onEntitySpawn(GameEvent<GameEventArgs.EntitySpawnEvent> e) {
     private boolean checkPreventEntitySpawn(GameEvent<GameEventArgs.EntitySpawnEvent> e, Location entityLocation) {
         CreatureSpawnEvent.SpawnReason spawnReason = e.getArgs().spawnReason;
 
+        if (BEE_ENTITY_TYPE != null &&
+                e.getArgs().entity.getType() == BEE_ENTITY_TYPE &&
+                BEEHIVE_SPAWN_REASON != null &&
```

Comment:
No need to check if it's null, only if `spawnReason == BEEHIVE_SPAWN_REASON` (spawnReason will never be null)